### PR TITLE
Validation 7.0

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[Validation 7.0]

### Issues Resolved
[1]

### Check List
- [ ] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,30 @@ HPE OneView makes it simple to deploy and manage today’s complex hybrid cloud
...
+You can find the latest supported HPE OneView Terraform Provider SDK [here](https://github.com/HewlettPackard/terraform-provider-oneview/releases/tag/v7.0.0-13)
 
 ## What's New
 
-HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 3800 (OneView v6.6)
+HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 4000 (OneView v7.0)
 
 Please refer to [notes](https://github.com/HewlettPackard/terraform-provider-oneview/blob/master/CHANGELOG.md) for more information on the changes , features supported and issues fixed in this version
+### Migration to HPE OneView 7.0 from HPE OneView 6.x ( Non-Synergy appliance )
+With HPE OneView 7.0 one can migrate your server hardware from HPE OneView 6.x using migrate functionality provided in the UI.
```
Comment (AvijitOS) [inline README.md:25]: one can migrate his/her server hardware

File: README.md [hunk 2]
```diff
@@ -14,13 +14,30 @@ HPE OneView makes it simple to deploy and manage today’s complex hybrid cloud
...
+#### Impact after migration
+1. Idempotency issues post migration.
+Post migration do run terraform refresh to sync with migrated resources.
+
+    Steps: 
+    -  Point your <TF_VAR_endpoint> to HPE OneView 7.0.
+    -  Run `terraform refresh` to sync with  migrated resource 
+  
+2. Idempotency issue in server profile after migration.
+With HPE OneView 7.0 the server hardware type field is not compatible with HPE OneView 6.x server hardware type. To resolve that, while creating server profile, make sure terraform configuration file is not having server hardware type field defined .
```
Comment (AvijitOS) [inline README.md:37]: In HPE OneView 7.0, the server hardware type ....

File: README.md [hunk 3]
```diff
@@ -14,13 +14,30 @@ HPE OneView makes it simple to deploy and manage today’s complex hybrid cloud
...
 ## What's New
 
-HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 3800 (OneView v6.6)
+HPE OneView Terraform SDK library extends support of the SDK to OneView REST API version 4000 (OneView v7.0)
 
 Please refer to [notes](https://github.com/HewlettPackard/terraform-provider-oneview/blob/master/CHANGELOG.md) for more information on the changes , features supported and issues fixed in this version
+### Migration to HPE OneView 7.0 from HPE OneView 6.x ( Non-Synergy appliance )
+With HPE OneView 7.0 one can migrate your server hardware from HPE OneView 6.x using migrate functionality provided in the UI. 
+With server hardware migration , corresponding server profile will also get migrated to HPE OneView 7.0 . 
+
```
Comment (AvijitOS) [inline README.md:27]: At the end, may be we can add "For more details one need to check HPE OneView documentation on migration supportability"

File: README.md (hunk 2 above)
Comment (nabhajit-ray) [reply README.md]: Done

**Pull request merged**